scripts/vale-review.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# This runs Vale on updated files, checks if the Vale alerts are on new/modified lines, and if so, builds curl request for GitHub review comment API
```

Comment:
Cna we also add a line that in order to test this locally, you must `export $GITHUB_AUTH_TOKEN`